templates/macros.html

```diff
@@ -26,6 +26,21 @@
         type="text/css" media="all" />
 {% endmacro highlight_css %}
 
+{#
+    Creates a list entry for active tabs. When the active tab is the same as `expected`, it will show the current tab as active.
+    * `expected` A string that represents the current tab, when `active_tab == expected` the current will be shown as active
+    * `href` A string used as the tab's link
+    * `text` A string used as the tab's text
```

Comment:
Make sure to note that `text` won't be escaped as html
```suggestion
    * `text` A string used as the tab's text, will *not* be escaped in any way
```